### DISCUSSION TITLE
How to use eCapture on Amazon linux 2

### DISCUSSION THREAD

**Discussion by tkumark** [gojue/ecapture#509] [General]
I want to capture incoming and out going traffice of a container running on EC2 Amazon Linxu2. Any help would be great. I tried tcp dump. I am able to capture incoming call but out going since they are https I am not able to see the details. I am new to ebpf and ecapture. 
when I treid to run ./ecapture tls  I got the following error

```
tls_2024/03/07 08:59:54 ECAPTURE :: ecapture Version : linux_x86_64:v0.7.5:5.15.0-1057-azure
tls_2024/03/07 08:59:54 ECAPTURE :: Pid Info : 4848
tls_2024/03/07 08:59:54 ECAPTURE :: Kernel Info : 5.4.255
2024/03/07 08:59:54 read keylogger :ld.so.conf.d/*.conf error .
tls_2024/03/07 08:59:54 EBPFProbeOPENSSL        module initialization failed. [skip it]. error:cant found openssl so load path
tls_2024/03/07 08:59:54 ECAPTURE ::     No runnable modules, Exit(1)
```

**Comment by cfc4n** [comment]
ecapture tls --libss=/path/xxxxxxx/libssl.so

exec ecapture tls --help for more detail.
